internal/cloudapi/v2/v2.go

```diff
@@ -92,6 +112,30 @@ func (s *Server) IncRequests(next echo.HandlerFunc) echo.HandlerFunc {
 	}
 }
 
+func (s *Server) ValidateRequest(next echo.HandlerFunc) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		request := c.Request()
+
+		route, params, err := s.router.FindRoute(request)
+		if err != nil {
+			return err
```

Comment:
These errors shold be defined in https://github.com/osbuild/osbuild-composer/blob/main/internal/cloudapi/v2/errors.go.

Reply:
ErrorFailedToOpenAPISpec makes sense for GetSwagger, but what would be appropriate for validate/new router errors?